.github/workflows/build.yml

```diff
@@ -11,10 +11,10 @@ jobs:
         include:
          - os: windows-latest
            dist: 'dist'
-           installer: 'dist/Better-CrewLink Setup*.exe'
+           installer: 'dist/BetterCrewLink Setup*.exe'
          - os: ubuntu-latest
            dist: 'dist:linux'
-           installer: 'dist/Better-CrewLink-*.AppImage'
```

Comment:
You can't change the installer name here, that is based of the electron-builder.yml but changing things there will frick the updater.